feat: add flagsmith support to the playground (#80)
open-feature/playground: README.md

Now that everything is configured, you should be able to [start the demo](#how-to-run-the-demo). Once it's started, select `launchdarkly` from the provider list located at the bottom right of your screen. You should now be able to control the demo app via LaunchDarkly!

### Flagsmith

> [Flagsmith](https://flagsmith.com/) is a open-source feature flag and remote config service. They offer many deployment options including: SaaS, private cloud, or on-prem.

Reviewer: ```suggestion
[Flagsmith](https://flagsmith.com/) is an open-source feature flag and remote config service. Flagsmith offers many deployment options including: SaaS, private cloud, and on-premise.
```
